draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4059,7 +3762,7 @@ role. The content of the attribute is a 64-bit unsigned integer in
 network byte order, which contains a random number. The number is used
 for solving role conflicts, when it is referred to as the tie-breaker
 value. An ICE agent MUST use the same number for all Binding requests,
-for all streams, within an ICE session. The ICE agent MAY change
+for all streams, within an ICE session. The ICE agent MUST NOT change
```

Comment:
This seems like it will not work well with 5245 things. 5245 says "An ICE restart (Section 9.1) causes a new selection of roles and tie-breakers."